pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -294,6 +306,102 @@ def simulate_and_vjp(
 # The set of supported observables.
 
 
+def stopping_condition(op: Operator) -> bool:
+    """A function that determines whether or not an operation is supported by ``lightning.kokkos``."""
+    # These thresholds are adapted from `lightning_base.py`
+    # To avoid building matrices beyond the given thresholds.
+    # This should reduce runtime overheads for larger systems.
+    if isinstance(op, qml.QFT):
+        return len(op.wires) < 10
+    if isinstance(op, qml.GroverOperator):
+        return len(op.wires) < 13
+
+    # As ControlledQubitUnitary == C(QubitUnitrary),
+    # it can be removed from `_operations` to keep
+    # consistency with `lightning_qubit.toml`
```

Comment:
Do we want to keep this comment?

Reply:
I think now it is not necessary. I will delete it :+1: